modules/ROOT/pages/activity-tracking-edit.adoc

```diff
@@ -0,0 +1,58 @@
+= Activity Tracking
+
+B2B transactions often communicate contractual obligations between companies. Due to the sensitivity of this information, it is desirable (and often necessary) to track transmissions and store both transmission content and metadata.
+
+Anypoint Partner Manager enables you to monitor trading partner transmission activity. Activity is the complete set of end-to-end events related to transmission metadata, messages, and errors that are involved in processing a transmission.
+
+Metadata includes things like the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+
+Common uses for this metadata include:
+
+* Nonrepudiation
+* Auditing
+* Correlation of acknowledgments
+* Error reporting
+
+With activity tracking, you can:
+
+* Search for message metadata (such as partner name, message type, filename, control number, endpoint, or date received).
+* View and download the original payload related to the transmission.
+* Review errors and acknowledgment payloads.
+
+== Track Message Flow Activity
+
+To track message flow activity:
```

Comment:
This really doesn't add any value, because it repeats the title directly after the title.
The optimal setup, IMHO, is the title, a brief statement giving a chronological context to the task--maybe there are some prereqs or something you should do first, or a certain time when you want to perform this task?--and then the Steps subhead.
But this might all be for another day...I know you are operating under special circumstances, and I should prolly just cool it with the org. ideas....